contracts/Vault.vy

```diff
@@ -1155,7 +1159,7 @@ def expectedReturn(_strategy: address = msg.sender) -> uint256:
 
 
 @external
-def report(_return: uint256) -> uint256:
+def report(_gain: uint256, _loss: uint256) -> uint256:
```

Comment:
Gain is the wrong word to use. Gain implies that the number is profit. But that is not the case. The number is profit + debtOutstanding

Reply:
perhaps this should be revisited, have it be simply reported gains, but then add a `debtPayment` field too

Reply:
That would certainly make it easier for new users to understand.